pages/index.tsx

```diff
@@ -1,19 +1,231 @@
-import React from 'react';
-import { useGetAllUsersQuery } from '../generated/hooks';
+import React, { useState } from 'react';
+import {
+  Container,
+  Heading,
+  Box,
+  Grid,
+  Textarea,
+  Button,
+  Field,
+} from 'theme-ui';
 
-function Hello() {
-  const [{ data, fetching, error }] = useGetAllUsersQuery();
-
-  if (fetching) return <p>Loading...</p>;
-  if (error) return <p>Oh no... {error.message}</p>;
+function Placeholder(props) {
+  return (
+    <Box
+      sx={{
+        height: 50,
+        width: 125,
+        borderRadius: 4,
+      }}
+      p={2}
+      bg="indigo.2"
+      {...props}
+    >
+      <Box
+        sx={{
+          height: '6px',
+          width: '45px',
+          borderRadius: 2,
+        }}
+        mb={2}
+        bg="pink.1"
+      ></Box>
+      <Box
+        sx={{
+          height: '8px',
+          width: '66px',
+          borderRadius: 2,
+        }}
+        bg="pink.1"
+      ></Box>
+    </Box>
+  );
+}
+const boxStyles = {
+  border: '1px solid',
+  borderRadius: '4px',
+  borderColor: 'indigo.3',
+  boxShadow: '2px 4px 4px rgba(0, 0, 0, 0.25)',
+};
 
+function CreateLayout() {
+  const [formState, setState] = useState(() => ({
```

Comment:
create a hook for this expose the data and a setter fn which will spread the currentState and the passed in changes